.github/workflows/performance.yml

```diff
@@ -97,3 +97,11 @@ jobs:
                   name: failures-artifacts
                   path: ${{ env.WP_ARTIFACTS_PATH }}
                   if-no-files-found: ignore
+
+            - name: Run mobile performance tests
+              run: ./gutenberg/bin/reassure-tests.sh
+
+            - name: Report mobile performance results
+              run: yarn danger ci
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Note: if we ultimately decide to report the results in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5579 for the foreseeable, then we should remove this script altogether.

Reply:
I agree that using Gutenberg Mobile for reporting results will be sufficient for now until the depth of tests can be developed further. I've removed this code from the Performance workflow for this PR.